Kiosk/Admin/AdminCardTestingViewController.swift

```diff
@@ -38,7 +38,7 @@ class AdminCardTestingViewController: UIViewController {
                         return
                     }
 
-                    let cardDetails = "Card: \(card.name) - \(card.last4) \n \(card.cardToken)"
+                    let cardDetails = "Card: \(card.name ?? "") - \(card.last4 ?? "") \n \(card.cardToken ?? "")"
```

Comment:
The card should always be there, but it's an IUO and I just added `?? ""` to avoid SwiftLint warnings :)